pkg/azuredisk/azuredisk_v2.go

```diff
@@ -89,6 +92,31 @@ const (
 	azDriverNodeHealthy      azDriverNodeStatus = "Driver node healthy."
 )
 
+// Metrics subsystem and all of the keys used by the rest client.
+const (
+	RestClientSubsystem = "rest_client"
+	LatencyKey          = "throttling_request_latency_seconds" // edits: prefix or postfix
+)
+
+// LatencyAdapter implements LatencyMetric.
+type LatencyAdapter struct {
+	metric *metrics.HistogramVec
+}
+
+// Observe increments the request latency metric for the given verb/URL.
+func (l *LatencyAdapter) Observe(_ context.Context, verb string, u url.URL, latency time.Duration) {
+	l.metric.WithLabelValues(verb, u.String()).Observe(latency.Seconds())
+}
+
+// RateLimiterLatency reports the rate limiter latency in seconds per verb/URL.
+var rateLimiterLatency = metrics.NewHistogramVec(
+	&metrics.HistogramOpts{
+		Subsystem: RestClientSubsystem,
+		Name:      LatencyKey,
+		Help:      "Rate limiter latency in seconds. Broken down by verb and URL.",
+		Buckets:   []float64{0.1, .25, .5, 1},
```

Comment:
We've seen latencies greatly exceed 1s in the past. Perhaps add 5, 10, 15, 30, 45, 60?